htdocs/modulebuilder/template/myobject_list.php

```diff
@@ -536,8 +536,12 @@
 }
 
 $varpage = empty($contextpage) ? $_SERVER["PHP_SELF"] : $contextpage;
-$selectedfields = $form->multiSelectArrayWithCheckbox('selectedfields', $arrayfields, $varpage); // This also change content of $arrayfields
-$selectedfields .= (count($arrayofmassactions) ? $form->showCheckAddButtons('checkforselect', 1) : '');
+if (empty($conf->global->MAIN_CHECKBOX_LEFT_COLUMN)) {
```

Comment:
Should be if !empty
Default is current behaviour and option move column on left.

Reply:
Yes it's done